eo-runtime/src/test/java/org/eolang/PhDefaultTest.java

```diff
@@ -539,6 +634,26 @@ private static class Int extends PhDefault {
                     )
                 )
             );
+            this.add(
```

Comment:
@karishka1222 maybe we can create custom `Phi` instance for each test case instead of reusing `PhDefaultTest.Int`? Currently I see potential problems with sharing test state between multiple test cases